addons/im_livechat/static/tests/helpers/mock_server.js

```diff
@@ -186,7 +186,7 @@ MockServer.include({
             // defaults that would set it to true) and here operator and visitor
             // can't be differentiated in that regard.
             'is_pinned': false,
-            'members': members, // channel_partner_ids
+            'channel_partner_ids': channel_partner_ids,
```

Comment:
Omission on changing this, at first glance I thought it was `channelInfo` formatter but actually it's passed to `_mockCreate` so definitely needs conversion from `members` to `channel_partner_ids`,